### PR TITLE
feat(cloud): send email

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Implement send email on cloud: call admin tenant's connector to send.

## Testing

UTs and local tested, email received.

## Checklist

- [x] `.changset-staged`
- [x] unit tests
- [ ] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/cloud/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@logto/cli": "workspace:*",
+    "@logto/connector-kit": "workspace:1.0.0-rc.2",
```
Comment (gao-sun) [inline packages/cloud/package.json:27]:
```suggestion
    "@logto/connector-kit": "workspace:*",
```

File: packages/cloud/src/libraries/services.ts [hunk 2]
```diff
@@ -13,4 +19,80 @@ export class ServicesLibrary {
...
+              );
+              const connector = databaseConnectors.find((connector) => connector.id === id);
+
+              if (!connector) {
+                throw new RequestError(`Unable to find connector ${id}`, 500);
+              }
+
+              return connector.config;
+            }
+          );
```
Comment (gao-sun) [inline packages/cloud/src/libraries/services.ts:55]: you can use `trySafe` to wrap this section to make it cleaner

File: packages/cloud/package.json (hunk 1 above)
Comment (darcyYe) [inline packages/cloud/package.json:27]: It looks weird that lock the dependency to 1.0.0-rc.2 and hence refactored the @logto/connector-kit.
Maybe add a comments and update the dependency as soon as the new @logto/connector-kit release is available?

**Review by gao-sun — APPROVED** [review]

**Review by darcyYe — APPROVED** [review]

File: packages/cloud/package.json (hunk 1 above)
Comment (wangsijie) [reply packages/cloud/package.json:27]: updated to *

File: packages/cloud/src/libraries/services.ts (hunk 2 above)
Comment (wangsijie) [reply packages/cloud/src/libraries/services.ts:56]: updated

**Pull request merged**